frontend/src/components/pages/CardDetails/ProjectDetails.jsx

```diff
@@ -3,7 +3,7 @@ import { Paper, Button, Grid, Typography, TextField } from "@material-ui/core";
 import { makeStyles } from "@material-ui/core/styles";
 import ViewContentButton from "../../widgets/ViewContentButton";
 
-import RepositoryDetails from "../RepositoryDetails";
+import RepositoryDetails from "../../regions/RepositoryDetails";
```

Comment:
Can we move the RepositoryDetails component rather? Since it is only used inside CardDetails then the component files should be inside the CardDetails directory. 

This will make things more cohesive and easier to navigate. If something is used in only one place then it should be in that one place.

Reply:
Done.